spec/factories/users.rb

```diff
@@ -3,6 +3,7 @@
     email 'test@test.com'
     password '12345'
     password_confirmation '12345'
+    locale 'ru'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.